src/lib.rs

```diff
@@ -86,3 +87,76 @@ pub mod __macros_internal {
         pub last_ptr: u64,
     }
 }
+
+#[derive(Deref, DerefMut)]
+pub struct AppWrapper(send_wrapper::SendWrapper<bevy::app::App>);
+impl AppWrapper {
+    pub fn new() -> AppWrapper {
+        AppWrapper(send_wrapper::SendWrapper::new(bevy::app::App::default()))
+    }
+}
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash, Default)]
+pub struct HotReloadUpdate;
+
+pub trait ReloadableAppExt {
+    /// Call this with plugins and systems and it will auto-add and remove systems in the `Update` schedule to your running app
+    fn reloadable(
```

Comment:
I'd prefer `with_hot_patched` or something like that, similar to the old `with_children` API